x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -78,6 +79,7 @@ const createNewTrustedAppForOsScheme = <O extends OperatingSystem, E extends Con
         );
       },
     }),
+    version: forUpdateFlow ? schema.maybe(schema.string()) : schema.never(),
```

Comment:
@pzl FYI. I'm going to revert this back to the prior implementation, where the `version` property is not added to the `POST` case, in my next PR. Thats because we also derive TS Types from this schema which causes the `version` attribute to be defined there

see https://github.com/elastic/kibana/blob/1a2d7458806f2f49979aff7b0527f8829b179404/x-pack/plugins/security_solution/common/endpoint/types/trusted_apps.ts#L31-L32